plugins/agents/muninpluginrunner/exec.go

```diff
@@ -0,0 +1,74 @@
+package muninpluginrunner
+
+import (
+	"bufio"
+	"regexp"
+	"strconv"
+
+	"github.com/abrander/agento/plugins"
+	"github.com/abrander/agento/timeseries"
+)
+
+func init() {
+	plugins.Register("muninpluginrunner", newMuninPluginRunner)
+}
+
+// MuninPluginRunner will retrieve stub status.
+type MuninPluginRunner struct {
+	Cmd string `toml:"cmd" json:"cmd" description:"Command to run"`
+	Arg string `toml:"arg" json:"arg" description:"Arguments to command"`
+	kv  []KeyValue
+}
+
+type KeyValue struct {
```

Comment:
This should not be exported. Let's rename to `keyValue`. Or use a `map[string]float64` for the keyvalue store.

Reply:
@abrander fixed